client/src/config.c

```diff
@@ -495,6 +495,28 @@ bool config_load(int argc, char * argv[])
   }
   free(localFile);
 
+  // load config from XDG_CONFIG_HOME
+  char * xdgFile;
+  char * dir;
+
+  if ((dir = getenv("XDG_CONFIG_HOME")) != NULL)
+    alloc_sprintf(&xdgFile, "%s/looking-glass/client.ini", dir);
+  else if ((dir = getenv("HOME")) != NULL)
+    alloc_sprintf(&xdgFile, "%s/.config/looking-glass/client.ini", dir);
+  else
+    alloc_sprintf(&xdgFile, "%s/.config/looking-glass/client.ini", pw->pw_dir);
+
+  if (xdgFile && stat(xdgFile, &st) >= 0 && S_ISREG(st.st_mode))
```

Comment:
`S_ISLNK` allowed here too

Reply:
We are using `stat` and not `lstat`, and `stat` returns information for the destination of symlinks. I tested the code as is with symlinks and it loads the file.